x11-packages/cinnamon/build.sh

```diff
@@ -0,0 +1,59 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/linuxmint/cinnamon
+TERMUX_PKG_DESCRIPTION="Cinnamon shell"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="6.4.10"
+TERMUX_PKG_SRCURL="https://github.com/linuxmint/cinnamon/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=eff6d55ec2b6443cec5b829df6240b329fbfa77aafd2064a60fdde2fc5b59f34
+TERMUX_PKG_DEPENDS="glib, gobject-introspection, cjs, muffin, cinnamon-menus, dbus, gcr, gdk-pixbuf, atk, opengl, gtk3, pango, xapp, libx11, libxml2, intltool, sassc, cogl, clutter, clutter-gtk, gnome-backgrounds, cinnamon-control-center, python-pillow, python3-xapp, gettext, libadapta, mint-y-icon-theme, mint-themes, cinnamon-settings-daemon"
+TERMUX_PKG_BUILD_DEPENDS="g-ir-scanner, glib-cross"
+TERMUX_PKG_SUGGESTS="gnome-terminal, gnome-screenshot"
+TERMUX_PKG_PYTHON_COMMON_DEPS="pysass"
+TERMUX_PKG_VERSIONED_GIR=false
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-Ddocs=false
+-Dbuild_recorder=false
+-Ddisable_networkmanager=true
+-Dpy3modules_dir="$TERMUX_PYTHON_HOME/site-packages"
+-Dwayland=false
+-Dpolkit=false
+"
+
+termux_step_pre_configure() {
+	termux_setup_gir
+	termux_setup_glib_cross_pkg_config_wrapper
+
+	export TERMUX_MESON_ENABLE_SOVERSION=1
+}
+
+termux_step_post_make_install() {
+	# disabling this sections because they will not work in termux
+	mv $TERMUX_PREFIX/share/cinnamon/cinnamon-settings/modules/cs_user.py $TERMUX_PREFIX//share/cinnamon/cinnamon-settings/modules/cs_user.py.bak
+	rm -rf $TERMUX_PREFIX/share/cinnamon/applets/user@cinnamon.org
+	rm -rf $TERMUX_PREFIX/share/cinnamon/applets/network@cinnamon.org
+	rm -rf $TERMUX_PREFIX/share/cinnamon/applets/printers@cinnamon.org
+
+	########################################################################
+	# Based on :- https://src.fedoraproject.org/rpms/cinnamon/tree/rawhide #
+	########################################################################
+	# Install gschema overrides
+	schemas_dir="$TERMUX_PREFIX/share/glib-2.0/schemas"
+	mkdir -p "$schemas_dir"
+	install -Dm644 "$TERMUX_PKG_BUILDER_DIR/10_cinnamon-common.gschema.override" "$schemas_dir/10_cinnamon-common.gschema.override"
+	cat >"$schemas_dir/10_cinnamon-wallpaper.gschema.override" <<EOF
+[org.cinnamon.desktop.background]
+picture-uri='file://$TERMUX_PREFIX/share/backgrounds/gnome/adwaita-d.jpg'
+EOF
```

Comment:
Please keep the heredo style consistent.
```suggestion
	cat <<-EOF > "$schemas_dir/10_cinnamon-wallpaper.gschema.override"
		[org.cinnamon.desktop.background]
		picture-uri='file://$TERMUX_PREFIX/share/backgrounds/gnome/adwaita-d.jpg'
	EOF
```